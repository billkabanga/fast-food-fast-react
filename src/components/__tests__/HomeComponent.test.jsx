import React from 'react';
import { shallow } from 'enzyme';
import Home from '../HomeComponent';


describe('Home Page', () => {
  const wrapper = shallow(<Home/>);

  it('renders home page successfully', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
