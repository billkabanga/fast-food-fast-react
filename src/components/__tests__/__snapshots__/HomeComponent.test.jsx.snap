// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders home page successfully 1`] = `ShallowWrapper {}`;
